app/views/chats/index.html.erb

```diff
@@ -1,4 +1,7 @@
-<h2 id="create-target">CHAT</h2>
+<div class= "rectangle-title">
+  <button class= "back-button" data-chat-id=<%= chat.id %>><%= image_tag 'back'%></button>
```

Comment:
not sure if it make sense to have the data-chat-id in the back button. Maybe in a div that wraps all this.